soccer/Processor.cpp

```diff
@@ -386,6 +386,16 @@ void Processor::run() {
             delete packet;
         }
 
+        // Log referee data
+        vector<NewRefereePacket*> refereePackets;
+        _refereeModule.get()->getPackets(refereePackets);
+        fprintf(stderr, "Ref packets received this loop:%lu\n",
+                refereePackets.size());
+        for (NewRefereePacket* packet : refereePackets) {
+            SSL_Referee* log = _state.logFrame->add_raw_refbox();
+            log->CopyFrom(packet->wrapper);
+        }
```

Comment:
I have no idea how to fix this. I tried a few things, but couldn't get it to compile.

I would very much like some assistance with this tonight.